src/renderer/component/recommendedContent/view.jsx

```diff
@@ -37,14 +37,7 @@ export default class RecommendedContent extends React.PureComponent<Props, State
     const { claim, search } = this.props;
 
     if (claim && claim.value && claim.value.stream && claim.value.stream.metadata) {
-      const {
-        value: {
-          stream: {
-            metadata: { title },
-          },
-        },
-      } = claim;
-
+      const { title } = claim.value.stream.metadata;
```

Comment:
Flow had an issue with the destructuring. Not sure why it's ok with this

Reply:
This is much cleaner as well